base_report_to_printer/models/report.py

```diff
@@ -42,7 +25,8 @@ def print_document(self, cr, uid, ids, report_name, html=None,
             raise exceptions.Warning(
                 _('No printer configured to print this report.')
             )
-        return printer.print_document(report, document, report.report_type)
+        return printer.with_context(context).print_document(
+            report, document, report.report_type)
```

Comment:
Yeah I can probably migrate this one to v8. I know I ran into some serious issues with the `get_pdf` method due to the split in core Odoo, but we should be fine here.